### DISCUSSION TITLE
将绳网git后如何使用

### DISCUSSION THREAD

**Discussion by xiaozhu52055** [share121/inter-knot#3185] [其他]
![符玄](https://github.com/user-attachments/assets/7bdd8285-d8ca-49c7-9f60-4e024834bb59)

**Comment by xiaozhu52055** [comment]
绳网代码git后如何使用<img src="https://img-static.mihoyo.com/communityweb/upload/27fb9141bd6d9a72a5ebff8b6ae10a00.png" alt="黑塔 你说咩啊" width="75" >

**Comment by x97-1** [comment]
`没用`

**Comment by Pleasurecruise** [comment]
本地能跑，但没什么用。作者发git仓库一方面是为了生产协同，一方面是为了用开发者功能。如果真的想复刻一个，也没有那个人流，没什么必要。
